### ISSUE TITLE
Color contrast ratio of "Error message" text present for Combox is 2.85:1 which is less than 4.5:1:A11y_MSSQL for VSCode_User is Adding Connection_Adding connection via Parameters_ColorContrast.

### ISSUE THREAD

**Issue by ganagdhar123** [microsoft/vscode-mssql#19815]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."
### GitHub Tags:
#A11yTCS;#MSSQL for VSCode;#SH_MSSQLForVSCode_Win32_Jul2025;#DesktopApp;#FTP;#Win11;#A11ySev2;#WCAG1.4.3;#A11yMAS;
### Environment Details:
Application: MSSQL for VSCode
Visual studio code version: 1.102.1
OS: Windows 11 Enterprise 24H2
Build: 26100.4652
### Repro Steps:

1. Launch visual studio code application
2. Sign in with "GitHub" account
3. Tab till "Extension" and press enter
4. Tab till search box and enter "SQL" (mssql) server and install extension
5. Tab till "Add connection" and press enter "connect to database" screen will get open.
6. Tab till input type and select parameter
7. Fill the all required fields in connect to database dailogue.
8. Tab till connect and press enter.
9. Observe that Color contrast ratio of "Error message" text present for Combox is 2.85:1 which is less than 4.5:1:

### Actual:
Color contrast ratio of "Error message" text present for Combox is 2.85:1 which is less than 4.5:1
### Expected:
Color contrast ratio of "Error message" text present for Combox should be greater than or equals to 4.5:1
### User Impact:
Users with visual impairments may not be able to perceive the error message, leading to confusion, missed information, and potential form submission errors.
### Attachment:

![Image](https://github.com/user-attachments/assets/cca84aec-01c8-4111-a44e-6ce9c20091d4)

**Comment by Benjin** [comment]
This should be addressed in our next release (1.35).

**Comment by Yash14j** [comment]
Issue is fixed in Version: 1.106.1 (user setup)
hence closing the bug

<img width="976" height="580" alt="Image" src="https://github.com/user-attachments/assets/8b489ca5-5f6b-44d0-af50-e1d4ac712f07" />

**Comment by Yash14j** [comment]
#A11yFixed;#A11yClosed;
